### ISSUE TITLE
convenient interface to the cdf of given distribution

### ISSUE THREAD

**Issue by davidovitch** [jonathf/chaospy#30]
I am trying to figure out how to get the cdf from a given distribution. For a pdf the interface is straight forward:
```
import numpy as np
import chaospy as cp
rayleigh = cp.Rayleigh(scale=11)
ws = np.arange(0, 25, 0.1)
rayleigh.pdf(ws)
```
However, a similarly convenient interface for the cdf is not available (or is it?). Is this a deliberate choice, or is it something that remains to be implemented? Is there an obvious alternative that would give me the cdf for a given range of values that I am missing? I am a little confused here because I can clearly see that the cdf for each distribution is defined in the ```_cdf``` method in ```chaospy.dist.cores```.

**Comment by jonathf** [comment]
The cumulative distribution function is available under the name `fwd`.

The reason for the name confusion is that the `fwd` represents forward Rosenblatt transformations, not CDF. In 1D they are the same, but in higher dimensions they have a quite different interpretation.

The reason for the `_cdf` is historical, and correctly as you point out is the CDF. But `_cdf` is also meant as a 1D contraption, so it wouldn't be incorrect to call it that.

Hope this answers your question.

**Comment by davidovitch** [comment]
Yes, thanks again. Maybe it is because I lack a solid understanding of this field, but do you think it would be helpful to include your note above in the doc string of ```chaospy.dist.backend.Dist.fwd```?

As a side note, can you recommend any good text book that covers (parts) of PCE, Rosenblatt transformations, distributions etc? Would that be [Handbook of Monte Carlo Methods](http://eu.wiley.com/WileyCDA/WileyTitle/productCd-0470177934.html) as mentioned in the [bibliography](http://chaospy.readthedocs.io/en/master/bibliography.html)?

**Comment by jonathf** [comment]
Adding the description in `fwd` is a good idea. I'll add it now so it will be included in the next release.

I would say that finding a textbook that covers all PCE, Rosenblatt and distributions does not exist. PCEs are too new and the literature is therefore somewhat limited.

Having said that, for understanding PCE I always recommend the excellent book by Dongbin Xiu:
http://press.princeton.edu/titles/9229.html

For distributions, the Handbook you note will do fine.

For the Rosenblatt transformations, it gets tricky. All I have learned comes from journal papers. My best answer is that I intend to add a section on the topic in my manual here:
http://chaospy.readthedocs.io/en/master/rosenblatt.html
So far it is empty, and it might take a few weeks until I get to it.

But if you are comfortable reading journal papers, I could recommend taking a look at the (open access) paper I wrote on the toolbox. It is somewhat short, but I address Rosenblatt transformations in section 3.1.:
http://dx.doi.org/10.1016/j.jocs.2015.08.008
